bot/exts/easter/earth_photos.py

```diff
@@ -0,0 +1,46 @@
+import aiohttp
+import discord
+from discord.ext import commands
+
+from bot.constants import Tokens
+
+UnClient_id = Tokens.unsplash_key
+
+
+class EarthPhotos(commands.Cog):
+    """This cog contains the command for earth photos."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        self.current_channel = None
+
+    @commands.command(aliases=["earth"])
+    async def earth_photos(self, ctx: commands.Context) -> None:
+        """Returns a random photo of earth, sourced from Unsplash."""
+        async with ctx.typing():
+            async with aiohttp.ClientSession() as session:
+                async with session.get(
+                        'https://api.unsplash.com/photos/random?query=earth&client_id=' + UnClient_id) as r:
```

Comment:
We should pass params using a dict and the `params` kwarg with aiohttp, rather than adding directly to the url.